test/models/department_test.rb

```diff
@@ -2,13 +2,12 @@
 
 class DepartmentTest < ActiveSupport::TestCase
   def setup
-    @department = Department.new
+    @department = create :department
   end
 
-  test 'department is saveable with user relation' do
-    refute @department.valid?
-    @department.contact = build :contact
+  test 'department is valid' do
     assert @department.valid?
-    assert @department.save!
+    @department.contact.name = ''
```

Comment:
your right, but not. It should go, but it shouldn't be in this model test at all. Plus I've allready tested for this in contact_test before. And thats where it should be, as it is a contact, not a department concern. Thx.